packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx

```diff
@@ -0,0 +1,121 @@
+import { useMemo, useState } from 'react'
+
+import { FlexBox } from '@/components/FlexBox/FlexBox'
+import { Information } from '@/components/Information'
+import { JoyTokenIcon } from '@/components/JoyTokenIcon'
+import { NumberFormat } from '@/components/NumberFormat'
+import { Text } from '@/components/Text'
+import { TextButton } from '@/components/_buttons/Button'
+import { FormField } from '@/components/_inputs/FormField'
+import { TokenInput } from '@/components/_inputs/TokenInput'
+import { DetailsContent } from '@/components/_nft/NftTile'
+import { DialogModal } from '@/components/_overlays/DialogModal'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+
+export type SellTokenModalProps = {
+  tokenId: string
+  onClose: () => void
+}
+
+const getTokenDetails = (_: string) => ({
+  title: 'JBC',
+  pricePerUnit: 1000,
+  tokensOnSale: 67773,
+  userBalance: 100000,
+})
+
+export const SellTokenModal = ({ tokenId, onClose }: SellTokenModalProps) => {
+  const [tokens, setTokens] = useState<number | null>(null)
+  const smMatch = useMediaMatch('sm')
+  const { pricePerUnit, tokensOnSale, userBalance, title } = getTokenDetails(tokenId)
+
+  const details = useMemo(
+    () => [
+      {
+        title: 'You will get',
+        content: (
+          <NumberFormat
+            value={tokens || 0}
+            as="p"
+            variant="t200"
+            withDenomination="before"
+            withToken
+            customTicker="$JBC"
+          />
+        ),
+        tooltipText: 'Lorem ipsum',
+      },
+      {
+        title: 'Fee',
+        content: <NumberFormat value={tokensOnSale} as="p" variant="t200" withDenomination="before" withToken />,
+        tooltipText: 'Lorem ipsum',
+      },
+      {
+        title: 'You will spend',
+        content: <NumberFormat value={tokensOnSale} as="p" variant="t200" withDenomination="before" withToken />,
+        tooltipText: 'Lorem ipsum',
+      },
+    ],
+    [tokens, tokensOnSale]
+  )
+  return (
+    <DialogModal
+      title="Sell $JBC"
+      show
+      onExitClick={onClose}
+      primaryButton={{
+        text: 'Sell tokens',
+      }}
+    >
+      <FlexBox flow="column" gap={8}>
+        <FlexBox gap={6} equalChildren>
+          <DetailsContent
+            avoidIconStyling
+            tileSize={smMatch ? 'big' : 'bigSmall'}
+            caption="PRICE PER UNIT"
+            content={pricePerUnit}
+            icon={<JoyTokenIcon size={smMatch ? 24 : 16} variant="silver" />}
+            withDenomination
+          />
+          <DetailsContent
+            avoidIconStyling
+            tileSize={smMatch ? 'big' : 'bigSmall'}
+            caption={`YOUR $${title} BALANCE`}
+            content={userBalance}
+            icon={<JoyTokenIcon size={smMatch ? 24 : 16} variant="silver" />}
```

Comment:
```suggestion
```

But also if I understand correctly `userBalance` is in $JBC but the $USD conversion is based on the current $JOY/$USD rate so the $USD value will be wrong.

Reply:
I made it a separate issue #4761